Content.Server/Nutrition/EntitySystems/SmokingSystem.cs

```diff
@@ -35,13 +35,15 @@ public sealed partial class SmokingSystem : EntitySystem
         [Dependency] private readonly ForensicsSystem _forensics = default!;
 
         private const float UpdateTimer = 3f;
+        private const float SmokableDuration = 360f;
 
         private float _timer;
 
         /// <summary>
         ///     We keep a list of active smokables, because iterating all existing smokables would be dumb.
         /// </summary>
         private readonly HashSet<EntityUid> _active = new();
+        private readonly Dictionary<EntityUid, FixedPoint2> _initialVolume = new();
```

Comment:
This isn't going to serialize properly so you can't use this.

Reply:
> This isn't going to serialize properly so you can't use this.

I can change this to work a different way, can you tell me why it wont serialize properly? Not very C# literate and it never threw any errors in testing that I could see, this would be my first contribution to anything as well.